cfme/common/__init__.py

```diff
@@ -236,15 +236,22 @@ class SummaryTable(object):
 
     MULTIKEY_LOC = '../../../tbody/tr[1]/td/strong'
 
-    def __init__(self, o, text, entry):
+    def __init__(self, o, text, entry, skip_load=False):
         self._object = o
         self._text = text
         self._entry = entry
         self._raw_keys = []
         self._keys = []
-        self.load()
+        self._multitable = False
```

Comment:
@RonnyPfannschmidt OK, this is what I am planning to to, I will add warning message once for each parent table (for each child it will be too much, in cases of table with many lines), and i will add massage to cfme.log  for each child table.

The log massage will add to the "SummaryTable.__init__"  and the console warning will add to "SummaryTable.load" (on line 264)

What do you think?